packages/typescript/src/components/Interface.tsx

```diff
@@ -41,29 +49,38 @@ export interface InterfaceMemberProps {
   children?: Children;
   optional?: boolean;
   readonly?: boolean;
+  refkey?: Refkey;
 }
 
 /**
  * Create a TypeScript interface declaration.
  */
 export function InterfaceMember(props: InterfaceMemberProps) {
   const namer = useTSNamePolicy();
+  const name = namer.getName(props.name ?? "", "interface-member");
+
   const type = props.type ?? props.children;
   const optionality = props.optional ? "?" : "";
   const readonly = props.readonly ? "readonly " : "";
+
   if (props.indexer) {
     return (
       <>
         [{props.indexer}]: {type}
       </>
     );
   } else {
-    return (
+    const memberContent = (
       <>
         {readonly}
-        {namer.getName(props.name!, "interface-member")}
```

Comment:
Now that symbols are being created, the name needs to use the `MemberName` component (because it may be deconflicted with other symbols). But don't worry about this, I'm adding this part as part of the member name escaping PR.